app/src/main/java/com/example/kuba10/mypokemonplaces/ChooseFragment/ChooseFragment.java

```diff
@@ -1,98 +0,0 @@
-package com.example.kuba10.mypokemonplaces.ChooseFragment;
-
-
-import android.content.Context;
-import android.os.Bundle;
-import android.support.v4.app.Fragment;
-import android.support.v7.widget.GridLayoutManager;
-import android.support.v7.widget.RecyclerView;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.LinearLayout;
-
-import com.example.kuba10.mypokemonplaces.Constants;
-import com.example.kuba10.mypokemonplaces.FragmentListener;
-import com.example.kuba10.mypokemonplaces.Model.PokemonGo;
-import com.example.kuba10.mypokemonplaces.R;
-
-import java.util.ArrayList;
-
-
-public class ChooseFragment extends Fragment {
-
-
-    private PokemonImageAdapter pokeAdapter;
-    private GridLayoutManager gridManager;
-    private RecyclerView recyclerView;
-    private LinearLayout errorImage;
-    private FragmentListener fragmentListener;
-    private ArrayList<PokemonGo> pokemonGo_data_list;
-
-
-    public static ChooseFragment newInstance(ArrayList<PokemonGo> pokemonGo_data_list) {
-        ChooseFragment fragment = new ChooseFragment();
-
-        Bundle args = new Bundle();
-        args.putParcelableArrayList(Constants.LIST, pokemonGo_data_list);
-        fragment.setArguments(args);
-        return fragment;
-
-
-    }
-
-    @Override
-    public void onAttach(Context context) {
-        super.onAttach(context);
-
-        fragmentListener = (FragmentListener) context;
-        pokemonGo_data_list = getArguments().getParcelableArrayList(Constants.LIST);
```

Comment:
Jak wyżej: zebranie argumentów gdzie indziej, + asserscja przy castowaniu

Reply:
Więc... co właściwie powinno się znajdować w onAttach?

Reply:
Tylko to co dotyczy podłączania `Fragment` do `Activity`
https://developer.android.com/reference/android/app/Fragment.html#onAttach(android.content.Context)